Dnd/DndStyleSheet.css

```diff
@@ -0,0 +1,28 @@
+body {
```

Comment:
Instead of setting this on `body`, I would set it on `dnd-bg` (as that's the CSS class on `body`)

Reply:
This was a mistake on my part, I thought I got rid of dnd-bg. If I remove dnd-bg is it then okay to put it straight onto body? I'm trying to limit how many classes I make that's all.

Reply:
It's better to have more CSS classes and show intent than place it on `body` directly imo.